app/javascript/controllers/autoCalibration.js

```diff
@@ -0,0 +1,114 @@
+/*
+    WebPlotDigitizer - https://automeris.io/WebPlotDigitizer
+    Copyright 2010-2019 Ankit Rohatgi <ankitrohatgi@hotmail.com>
+    This file is part of WebPlotDigitizer.
+    WebPlotDigitizer is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    WebPlotDigitizer is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+    You should have received a copy of the GNU Affero General Public License
+    along with WebPlotDigitizer.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+var wpd = wpd || {};
+wpd.autoAlign = (function() {
+       
+    function number_detection(resolve){
+        let img =wpd.graphicsWidget.getImagePNG();
+
+        var det_point = {
+            x: new Array(), 
+            y: new Array(), 
+            val: new Array(),
+        };         
+            Tesseract.recognize(img, {
```

Comment:
Hey, I tried using this a couple of days ago, but looks like this patch isn't complete. I was able to load Tesseract, but didn't see anything happen on the default image. I'll give this another shot when I have more time, but if you happen to have a more complete patch then it'll save me some effort. Thanks!

Reply:
Hi, sorry that it didn't work. I just compiled it again it seems to work for me. There might be a difference as I am working with the mini version of tesseract (it was easier for me to import) so I just manually add this line in the compiled html offline file : 
`<script src='https://cdn.jsdelivr.net/gh/naptha/tesseract.js@v1.0.14/dist/tesseract.min.js'></script>`
But since tesseract loaded I doubt that this will be the problem. Do you get the console message : pre-main prep time:  12ms after running the auto align XY button and nothing comes next?